pipeline_dp/aggregate_params.py

```diff
@@ -160,23 +150,25 @@ def __post_init__(self):
             raise ValueError(
                 "AggregateParams.public_partitions is deprecated. Please use public_partitions argument in DPEngine.aggregate insead."
             )
-        if self.max_contributions is not None and self.max_contributions > 0:
+        if self.max_contributions is not None:
+            _check_positive_int(self.max_contributions, "max_contributions")
             if ((self.max_partitions_contributed is not None) or
                 (self.max_contributions_per_partition is not None)):
                 raise ValueError(
                     "Only one in params.max_contributions or "
                     "(params.max_partitions_contributed and "
                     "params.max_contributions_per_partition) must be set")
         else:
-            if ((self.max_partitions_contributed is None or
-                 self.max_partitions_contributed <= 0) and
-                (self.max_contributions_per_partition is None or
-                 self.max_contributions_per_partition <= 0)):
+            if (self.max_partitions_contributed is None or
+                    self.max_contributions_per_partition is None):
                 raise ValueError(
-                    "One amongst params.max_contributions or "
-                    "(params.max_partitions_contributed or "
-                    "params.max_contributions_per_partition) must be set to a "
-                    "positive value.")
+                    "Either params.max_contributions must be set or "
+                    "(params.max_partitions_contributed and "
```

Comment:
It would be nice if we got a separate error for these cases. ex. a user may just set `max_partitions_contributed` and not `max_contributions_per_partition`. Maybe raise a separate error?

Reply:
Thanks, good point. Done